apps/accounts/serializers.py

```diff
@@ -37,3 +40,46 @@ def update(self, instance, validated_data):
             profile.affiliation = affiliation
             profile.save()
         return instance
+
+
+class InviteUserToChallengeSerializer(serializers.ModelSerializer):
+    """
+    Serializer to store the invitation details
+    """
+
+    challenge_title = serializers.SerializerMethodField()
+    challenge_host_team_name = serializers.SerializerMethodField()
+    username = serializers.SerializerMethodField()
+
+    class Meta:
+        model = InviteUserToChallenge
+        fields = (
+            "email",
+            "invitation_key",
+            "status",
+            "challenge",
+            "user",
+            "challenge_title",
+            "challenge_host_team_name",
+            "username",
+            "invited_by",
+        )
+
+    def get_challenge_title(self, obj):
+        return obj.challenge.title
+
+    def get_challenge_host_team_name(self, obj):
+        return obj.challenge.creator.team_name
+
+    def get_username(self, obj):
+        return obj.user.username
+
+
+class AcceptChallengeInvitationSerializer(serializers.ModelSerializer):
```

Comment:
We don't need this serializer. Can we please get rid of this and use the previous one?